iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/patent/PatentExporterJob.java

```diff
@@ -360,6 +377,7 @@ private static AtomicAction buildEntityAction(Patent patent, String patentIdToEx
         return OafProtos.Oaf.newBuilder()
                 .setKind(KindProtos.Kind.entity)
                 .setEntity(buildOafEntity(patent, patentIdToExport, patentDateOfCollection, patentEpoUrlRoot))
+                .setDataInfo(buildOafEntityDataInfo())
```

Comment:
OK, I have overlooked it while reviewing but noticed it when rebasing and aligning this change to a new model: we could rely on a static `OAF_ENTITY_DATAINFO` value here, right?

Reply:
Yes, I must have forgot to replace it with this static value. Created a new PR to merge this change #1062 .